source/developers/references/developer_guide/process/contributor.rst

```diff
@@ -1,47 +1,75 @@
-.. _Contributor:
-
 ************
 Contributing
 ************
 
+Contributing your custom code to the Open edX project is a two-step process:
```

Comment:
@crathbun428 - would you mind reviewing this file? It's documentation (developer-oriented) about how to use feature requests.

Reply:
You can see it rendered here: https://docsopenedxorg--395.org.readthedocs.build/en/395/developers/references/developer_guide/process/contributor.html

Reply:
Looks good to me, a couple of suggested edits updated text below:
1. In paragraph 1: Contributing your custom code to the Open edX project is a two-step process: review of your idea by the **Product Working Group** (the Product Review Process), followed by a review of your code by the project’s core committers. We describe this process on this page. <--- I'm assuming we'd have a link to [the wiki page](https://openedx.atlassian.net/wiki/spaces/COMM/overview) here?

The rest looked good to me!

Reply:
Yes, I totally forgot to add the link. Thank you!